CockpitPlugin/build.gradle

```diff
@@ -1,13 +1,14 @@
 buildscript {
     ext.kotlin_version = '1.2.51'
+    ext.cockpit_core_version = '1.1.1-SNAPSHOT'
```

Comment:
What about extracting `cockpit_core_version` to `dependecies.gradle`?
`ext {
    ext.cockpit_core_version = '1.1.1-SNAPSHOT'
}`

Reply:
Unfortunately we can't use:
```
apply from: files('../gradle/dependencies.gradle')
```
in `CockpitPlugin/build.gradle` because of: 
```
only buildscript {} and other plugins {} script blocks are allowed before plugins {} blocks, no other statements are allowed
```